examples/data-objects/codemirror/src/codeMirrorView.tsx

```diff
@@ -46,7 +46,7 @@ class CodeMirrorView {
 		// https://stackoverflow.com/questions/18828658/how-to-kill-a-codemirror-instance
 
 		if (this.sequenceDeltaCb) {
-			this.text.removeListener("sequenceDelta", this.sequenceDeltaCb);
+			this.text.off("sequenceDelta", this.sequenceDeltaCb);
```

Comment:
This used to leak the underlying event implementation, which had more methods. removeListener is an alias for off, which is the only option now that this is going through the interfaces not exposing the underlying class.

Reply:
(According to the Node.js docs `off` is an alias for `remvoeListener`, but, yeah, we promote use of `off`; so, cool.)